docs/tutorials/01_basic_example.ipynb

```diff
@@ -655,7 +638,7 @@
    "id": "916e166e-3e89-4201-b24e-7b3802a97dae",
    "metadata": {},
    "source": [
-    "Now it would be good to know what is the cost when we subsitute some numbers. We can do this using `evaluate` method. As you can see in the example below, it can either take all the parameters or just some of them."
+    "Now it would be good to know what is the cost when we subsitute some numbers. We can do this using `evaluate` method. As you can see in the example below, it can either take substitute all the parameters or just some of them."
```

Comment:
There is something wrong with last sentence, I think we should delete "take".

```suggestion
    "Now it would be good to know what is the cost when we subsitute some numbers. We can do this using `evaluate` method. As you can see in the example below, it can either substitute all the parameters or just some of them."
```